scripts/msys2-geany-release.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+
+import glob
+import os
+import shutil
+from os.path import exists, isfile, join
+from subprocess import check_call
+import re
+
+"""
+This script prepares a Geany release on Windows.
+The following steps will be executed:
+- update geany-themes repository
```

Comment:
Not strictly related to this PR but I don't want to get it missed before the release:

Thanks to #4044, we don't need to manually pull the geany-themes files any longer as (most) they are already included in Geany.

If you like, just drop the relevant code completely.

Reply:
Ok I will apply the change later today or tomorrow

Reply:
done